### ISSUE TITLE
Combo Actions - Outgoing Message 1 click Finish

### ISSUE THREAD

**Issue by ProstheticAIM** [Exslims/MercuryTrade#254]
Can you make a way to combo things like finishing the trade i can click one button and it does multiple actions
Click "Thank you" -> Sends "Thanks for the trade, GL" -> /Kick %self% -> /Hideout

**Comment by Krytos** [comment]
Yes, this is what's keeping me with TradesCompanion. We already have a checkbox to close the window with custom messages. Just add the option to leave the party as well.

**Comment by TemporalWolf** [comment]
Given that would make Mercury Trade violate the "a macro can only result in a single action" rule, it would then be bannable to use MT... so I'm definitely against this.
